### PR TITLE
[Feature][JsonSplit-api] define task api

### PR DESCRIPTION
**Pull request by brave-lee**
api module #5498

### CODE REVIEW

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/TaskDefinitionServiceImpl.java [hunk 1]
```diff
@@ -277,5 +277,30 @@ public void checkTaskNode(Map<String, Object> result, TaskNode taskNode, String
         putMsg(result, Status.SUCCESS);
         return result;
     }
+
+    @Override
+    public Map<String, Object> queryTaskDefinitionVersions(User loginUser, String projectName, int pageNo, int pageSize, long taskCode) {
+        return null;
```
Comment (wen-hemin) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/TaskDefinitionServiceImpl.java:283]: Is this to be implemented later?

**Review by wen-hemin — COMMENTED** [review]
Please check.

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/TaskDefinitionServiceImpl.java (hunk 1 above)
Comment (brave-lee) [reply dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/TaskDefinitionServiceImpl.java:283]: yes, I think my students will finish it

**Comment by lenboo** [comment]
i noticed that this pr have not any relation with others module,  i suggest that you can submit this pr to branch 'dev'.
of course, it's up to you.

**Review by wen-hemin — APPROVED** [review]
+1

**Pull request merged**